src/main/java/eutros/multiblocktweaker/crafttweaker/construction/MultiblockBuilder.java

```diff
@@ -38,6 +42,8 @@
     public ResourceLocation loc;
     public int metaId;
     public float zoom = 1.0f;
+    public Map<ItemStack, List<ITextComponent>> tooltipMap = new HashMap<>();
```

Comment:
`ItemStack` doesn't override `Object#equals` and `Object#hashCode`, so this map stores its keys by identity. It should probably use the custom hashing strategy as in [GTCE's `MultiblockInfoPage`](https://github.com/GregTechCE/GregTech/blob/d9e7ed4570b990ed389a6a96589636ced78edd34/src/main/java/gregtech/integration/jei/multiblock/MultiblockInfoPage.java#L24). Otherwise, added multi-part tooltips may be nondeterministic since it's based on the hash map's iteration order.